Lib/test/test_fstring.py

```diff
@@ -361,6 +361,18 @@ def test_backslashes_in_string_part(self):
         self.assertEqual(f'2\x203', '2 3')
         self.assertEqual(f'\x203', ' 3')
 
+        with self.assertWarns(DeprecationWarning):
+            value = eval(r"f'\{6*7}'")
+        self.assertEqual(value, '\\42')
+        self.assertEqual(f'\\{6*7}', '\\42')
+        self.assertEqual(fr'\{6*7}', '\\42')
+
+        AMPERSAND = 123
```

Comment:
It took me literally an hour to figure out what's going on here.

Can you change this line to something:

    AMPERSAND = ord('{')  # =123: Make sure we get the right unicode character (&), and not pick up local variable